### ISSUE TITLE
PayPal cannot be tested in staging mode (SSL error)

### ISSUE THREAD

**Issue by ocram** [printdotio/pio.js#82]
When using your web widget in staging mode and choosing PayPal payment, it seems to call (or redirect to) `https://sandbox.paypal.com/cgi-bin/webscr`.

However, one cannot connect to that site because there's an SSL error. The certificate is only valid for `www.sandbox.paypal.com`, so one must probably just add a `www.` at the beginning of the host to make this work again.

**Comment by micahasmith** [comment]
surprisingly this is a paypal error (so far as we've investigated previously)

**Comment by ocram** [comment]
Yes, that's what I thought as well. It's probably true that PayPal had a valid certificate for the non-www domain in the past.
